### DISCUSSION TITLE
I found a fix for execution time icon in the Atomic theme

### DISCUSSION THREAD

**Discussion by coujean99** [JanDeDobbeleer/oh-my-posh#3438] [Show and tell]
Hello!

Recently I installed the prompt extention for the first time and everything works very well... Except one thing that didn't looks right.
After setting up the theme to be Atomic, I noticed that there is a small issue with the execution time segment: there is a free space in the label right before the end. I don't know if I'm the only one that encounted this problem nor if it has been issued.
![image](https://user-images.githubusercontent.com/54377330/216378007-6022652d-fad4-4f24-b0d0-cb989986fc24.png)

At first I was wondering "what is happening" without knowing if the problem was my computer (Canadian French), the program, the prompt or the font. Well I figured out that there is a space after the character.
![image](https://user-images.githubusercontent.com/54377330/216379135-975f3251-1314-4bb2-b3a8-e8c53e3906b2.png)

It's funny because the same character without sand (U+FBAB) has a bigger space but it don't shows up in the prompt.
<details>

![image](https://user-images.githubusercontent.com/54377330/216380921-fd9e13f5-0df9-4ed0-8dfb-1bc40002136b.png)
![image](https://user-images.githubusercontent.com/54377330/216380711-5be0d707-4dae-495c-a6d6-882c94123bed.png)

</details>

I don't know anything about how a font works so I can't blame it, however I tried if a "zero-width space" could fix the problem... and it does! I simply changed the space on line 62 for `\u200a` and voilà, it looks even better!
![image](https://user-images.githubusercontent.com/54377330/216383381-a26bc24b-06f1-4af4-951c-7e2cc207c745.png)
Here is the full string:
`"template": "\u200a\ufa1e{{ .FormattedMs }}\u2800"`

I didn't know where to share the fix without knowing if it affects others themes and segments. I'm only using k8 and Git for now (yeah I'm kinda new in this huge world 😁).

Have a nice day!

**Comment by JanDeDobbeleer** [comment]
@coujean99 great find, and we also have this documented in the FAQ if I'm not mistaken.

**Reply by coujean99** [reply]
If you are talking about the FAQ on your website, there is a section called "Unexpected space between segments/text" but the proposed solution is already implemented in the theme and is still not working for me 😕

**Reply by JanDeDobbeleer** [reply]
@coujean99 this is a bug report for Nerd Fonts and I'll also update the FAQ item with this information
